app/views/shf_application_mailer/acknowledge_received.html.haml

```diff
@@ -1,5 +1,20 @@
 - provide :signoff do
   = render 'application_mailer/signoff'
 
+- i18n_scope = 'mailers.shf_application_mailer.acknowledge_received'
 = render layout: 'layouts/mail_from_membership' do
-  %p= t('mailers.shf_application_mailer.acknowledge_received.message_text')
+  %p= t('message_text', scope: i18n_scope)
+
+  %hr
+  %p= t('your_submitted_info', scope: i18n_scope)
+  .shf-application-info
+    %p
+      = shf_app_to_html(@shf_application)
+  %hr
+
+  - if @shf_application.uploaded_files.size == 0
+    %p.nofiles-will-review= t('review_after_recevied_files', scope: i18n_scope)
+
+  %p.login-url
+    = 'You can log in and change your information and upload more files. '
```

Comment:
oh my got its not a t()

Reply:
oh the horror!!!

Reply:
@thesuss  - I've updated the locale info and even made this into a t() ;-)  
Ready for you to review it again to make sure I didn't blorf something up.

(then if all is ok, squash and merge it!)